### PR TITLE
aadding d2l-hm-constants-behavior to the map

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: dependency-map.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ d2l-colors,d2l-colors,BrightspaceUI/colors#semver:^4.0.0
 d2l-fastdom-import,fastdom,^1.0.8
 d2l-fetch,d2l-fetch,Brightspace/d2l-fetch.git#semver:^2.0.0
 d2l-hierarchical-view,d2l-hierarchical-view,BrightspaceUI/hierarchical-view#semver:^2.0.0
+d2l-hm-constants-behavior,d2l-hypermedia-constants,^6.0.0
```
Comment (awikkerink) [inline dependency-map.txt]: Sorry, just noticed this. This line will cause npm to look for the package in the npm registry.

```suggestion
d2l-hm-constants-behavior,Brightspace/d2l-hypermedia-constants,#semver:^6.0.0
```

Comment (awikkerink) [reply dependency-map.txt]: @dlockhart  I am too quick with the approves

Comment (awikkerink) [reply dependency-map.txt]: nvm. :sunglasses: https://www.npmjs.com/package/d2l-hypermedia-constants

Comment (dlockhart) [reply dependency-map.txt]: That's OK isn't it? It's published there:
https://www.npmjs.com/package/d2l-hypermedia-constants

Comment (awikkerink) [reply dependency-map.txt]: Yup. I didn't realise this was published

Comment (dlockhart) [reply dependency-map.txt]: No worries -- yeah I ended up needing to publish it because of the ES5 transpile step that's now required for use within NPM.

**Pull request merged**